BackofficeBundle/Controller/NodeController.php

```diff
@@ -31,8 +31,11 @@ class NodeController extends AbstractAdminController
     public function formAction(Request $request, $id)
     {
         $nodeRepository = $this->container->get('open_orchestra_model.repository.node');
-        $node = $nodeRepository->find($id);
+        $node = $nodeRepository->findVersionByDocumentId($id);
 
+        if (!$this->isGranted(TreeNodesPanelStrategy::ROLE_ACCESS_UPDATE_NODE, $node)) {
```

Comment:
why remove access when user haven't role ROLE_ACCESS_UPDATE_NODE ? how an user can see information of node if hasn't role. Normaly he can see a form but it is disabled. 

and this action is used also for the transveral node, so an user which hasn't role ROLE_ACCESS_UPDATE_NODE but he has role ROLE_ACCESS_UPDATE_NODE can update a transversal node